classes/haskell.bbclass

```diff
@@ -80,7 +80,10 @@ do_configure() {
     ghc-pkg recache
     ghc_version=$(get_ghc_version)
 
-    ${RUNGHC} Setup.*hs clean --verbose
+    if [[ -d "${S}/dist" ]]; then
```

Comment:
I think `[[` is a bash-ism, so we should just use single `[`.

Reply:
Fixed.